src/util/config.cpp

```diff
@@ -53,6 +53,7 @@ namespace util {
         maxInFlightRatio = this->getSystemConfIntParam("MAX_IN_FLIGHT_RATIO", "3");
         maxWorkersPerFunction = this->getSystemConfIntParam("MAX_WORKERS_PER_FUNCTION", "10");
         threadMode = getEnvVar("THREAD_MODE", "local");
+        ompThreadPoolSize = this->getSystemConfIntParam("OMP_THREAD_POOL_SIZE", "10");
```

Comment:
Set to 0 for no overhead on module start (I set it to 10 here because it needs to be on for tests, I could set it to 0 by default and change the env var in the CI instead

Reply:
Yes, I think the default configuration should be "all bells and whistles switched off", then they can be turned on when necessary.